ambry-store/src/test/java/com.github.ambry.store/HardDeleterTest.java

```diff
@@ -305,3 +306,76 @@ public void testHardDelete() {
     }
   }
 }
+
+class MockIndex extends PersistentIndex {
```

Comment:
Copied from PersistentIndexTest. Since PersistentIndexTest file is removed, adding it here